packages/clerk-js/src/ui/components/devPrompts/EnableOrganizationsPrompt/index.tsx

```diff
@@ -0,0 +1,247 @@
+              paddingBottom: t.space.$4,
+              gap: t.space.$2,
+            })}
+          >
+            <Flex
+              as='header'
+              sx={t => ({
+                display: 'flex',
+                alignItems: 'center',
+                gap: t.space.$2,
+              })}
+            >
+              <svg
+                width='20'
+                height='20'
+                viewBox='0 0 20 20'
+                fill='none'
+                xmlns='http://www.w3.org/2000/svg'
+              >
+                <path
+                  opacity='0.2'
+                  d='M17.25 10C17.25 14.0041 14.0041 17.25 10 17.25C5.99594 17.25 2.75 14.0041 2.75 10C2.75 5.99594 5.99594 2.75 10 2.75C14.0041 2.75 17.25 5.99594 17.25 10Z'
+                  fill='#EAB308'
+                />
+                <path
+                  fillRule='evenodd'
+                  clipRule='evenodd'
+                  d='M10 3.5C6.41015 3.5 3.5 6.41015 3.5 10C3.5 13.5899 6.41015 16.5 10 16.5C13.5899 16.5 16.5 13.5899 16.5 10C16.5 6.41015 13.5899 3.5 10 3.5ZM2 10C2 5.58172 5.58172 2 10 2C14.4183 2 18 5.58172 18 10C18 14.4183 14.4183 18 10 18C5.58172 18 2 14.4183 2 10Z'
+                  fill='#EAB308'
+                />
+                <path
+                  fillRule='evenodd'
+                  clipRule='evenodd'
+                  d='M10 6C10.5523 6 11 6.44772 11 7V9C11 9.55228 10.5523 10 10 10C9.44772 10 9 9.55228 9 9V7C9 6.44772 9.44772 6 10 6Z'
+                  fill='#EAB308'
+                />
+                <path
+                  fillRule='evenodd'
+                  clipRule='evenodd'
+                  d='M10 12C10.5523 12 11 12.4477 11 13V13.01C11 13.5623 10.5523 14.01 10 14.01C9.44772 14.01 9 13.5623 9 13.01V13C9 12.4477 9.44772 12 10 12Z'
+                  fill='#EAB308'
+                />
+              </svg>
+
+              <h1
+                css={css`
+                  ${basePromptElementStyles};
+                  color: white;
+                  font-size: 0.875rem;
+                  font-weight: 500;
+                `}
+              >
+                Organizations feature required
+              </h1>
+            </Flex>
+
+            <Flex
+              sx={t => ({
+                display: 'flex',
+                flexDirection: 'column',
+                gap: t.space.$0x5,
+              })}
+            >
+              <span
+                css={css`
+                  ${basePromptElementStyles};
+                  color: #b4b4b4;
+                  font-size: 0.8125rem;
+                  font-weight: 400;
+                  line-height: 1rem;
+                `}
+              >
+                To use the{' '}
+                <code
+                  css={css`
+                    ${basePromptElementStyles};
+                    color: white;
+                  `}
+                >
+                  {ctaText}
+                </code>{' '}
+                hook,
+                <br />
```

Comment:
Will want to avoid the br usage here, as at smaller breakpoints it leads to unintended breaks.

<img width="424" height="330" alt="Image" src="https://github.com/user-attachments/assets/ce559dd0-51c3-4ec5-8bd3-8652780ade59" />